cmd/create/oidcconfig/cmd.go

```diff
@@ -112,79 +128,172 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
-	oidcConfigStrategy, err := getOidcConfigStrategy(mode)
+	if !args.rawFiles {
+		r.Reporter.Infof("This command will create a S3 bucket populating it with documents " +
+			"to be compliant with OIDC protocol. It will also create a Secret in Secrets Manager containing the private key.")
+	}
+
+	if args.rawFiles && mode != "" {
```

Comment:
Maybe it is worth checking this before line `117` where we ask the user to choose the mode.
Then ask the user to choose a mode, only if `rawFiles` isn't supplied.